<launch>
  <node name="bluetooth_node" pkg="bluetooth_bridge" type="bluetooth_node.py" output="screen">  </node>
  <node name="serial_node" pkg="bluetooth_bridge" type="serial_node.py" output="screen">  </node>
  <node name="hilens_receiver" pkg="bluetooth_bridge" type="hilens_receiver.py" output="screen"></node>
  <node name="lane_detect" pkg="auto_driver" type="lane_detect.py" output="screen"></node>
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
    <param name="serial_port"         type="string" value="/dev/ttyUSB1"/>
    <param name="serial_baudrate"     type="int"    value="256000"/><!--A3 -->
    <param name="frame_id"            type="string" value="laser"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="scan_mode"           type="string" value="Sensitivity"/>
  </node>
  <node name="lidar_node" pkg="rplidar_ros" type="lidar_node.py" output="screen"></node>
</launch>
